docs/how-to/configure.md

```diff
@@ -196,38 +196,48 @@ Fill both fields and click the *Apply Alertmanager settings* button to proceed.
 
 ## Percona Platform
 
-This panel is where you create, and log into and out of your Percona Platform account.
+This panel is where you connect your PMM server to your Percona Platform Account.
 
-### Login
+!!! note alert alert-primary ""
+    Your Percona Platform Account is separate from your PMM User account.
 
-![!image](../_images/PMM_Settings_Percona_Platform_Login.jpg)
+### Connect PMM to Percona Platform
+Connect your PMM servers to Percona Platform to leverage Platform services that boost the monitoring capabilities of your PMM installations.
 
-If you have a *Percona Platform* account, enter your credentials and click *Login*.
+To connect to Percona Platform, you need a Percona Account. With a Percona Account you also get access to various Percona services, including Percona Platform, Percona Customer Portal, and Community Forum.
 
-Click *Sign out* to log out of your Percona Platform account.
+If you don't have a Percona Account, you can create one on the [Percona Account Sign Up](https://id.percona.com/signin/register/) page.
 
-### Sign up
 
-![!image](../_images/PMM_Settings_Percona_Platform_Sign_Up.jpg)
+#### Pre-requisites
+To ensure that PMM can establish a connection to Percona Platform: 
 
-To create a *Percona Platform* account:
+### Check that you are a member of an existing Platform organization
+1. Log in to [Percona Platform](https://portal.percona.com) using your Percona Account.
+2. On the *Getting Started page*, check that the *Create organization* step shows an option to view your organization.
 
-1. Click *Sign up*.
-2. Enter a valid email address in the *Email* field.
-3. Choose and enter a strong password in the *Password* field.
-4. Select the check box acknowledging our terms of service and privacy policy.
-5. Click *Sign up*.
+If you see an option to create a new organization instead, your Percona Account is not linked to any organization yet. If this is the case, contact your account administrator, or create a new organization for your Percona Account.
 
-A brief message will confirm the creation of your new account and you may now log in with these credentials.
+### Set the public address of your PMM server 
+1. In PMM, go to *Settings > Advanced Settings*.
+2. Enter your address/hostname or click *Get from browser* to enable your browser to automatically detect and populate this field.
+3. Save the changes.
 
-!!! note alert alert-primary ""
-    Your Percona Platform account is separate from your PMM User account.
+## Connect PMM to Percona Platform
+To connect your PMM server to Percona Platform:
+
+1. In PMM go to *Settings > Percona Platform* tab:
+    ![!image](../_images/PMM_Settings_Percona_Platform_Login.png)
+2. Fill in the *Connect PMM to Percona Portal* form with the name of your PMM instance and the credentials of your Percona Account:
+3. Click *Connect*.
+
+After connecting, you can also use SSO for PMM server. 
```

Comment:
Maybe we should also clarify that the PMM login screen will contain a separate  button `Sign in with Percona Account` ?

Reply:
But the current design is not what we want to have. it needs to be  revoked, maybe even for  2.26, so we'll change it  soon

Reply:
I'm just saying that we can point a user how and where he/she can use SSO login